msw/pd.nsi

```diff
@@ -144,6 +147,12 @@ SectionGroup /e "${PRODUCT_NAME}"
     SectionEnd
 
     Section "Create Startmenu entry" StartMenu
+        ; ugly hack so the app shows up in "recently added"
+        ${If} $CONTEXT == "all"
+            SetShellVarContext all
+        ${Else}
+            SetShellVarContext current
```

Comment:
can't we use `SetShellVarContext $CONTEXT` (also in L60 resp L65)

Reply:
I did try to use `SetShellVarContext $CONTEXT` (that was the initial intencion) but it didn't worked for me. I might have it wrong but it looks the compiler does not allow the expansion there? 

will investigate.

Reply:
not sure why but we can't:

```
Usage: SetShellVarContext all|current
Error in script "C:/msys64/tmp/tmp.qMn9ZCGmH5/pd.nsi" on line 156 -- aborting creation process
```

if i pop a messegebox i know `$CONTEXT` expands to `all`. the msgbox pops up almost at the end of the "copy files" part.

Reply:
> Don't worry. NSIS syntax is arcane...

good.